clang/tools/clang-linker-wrapper/ClangLinkerWrapper.cpp

```diff
@@ -772,7 +758,7 @@ readSYCLImagesFromTable(StringRef TableFile, const ArgList &Args) {
       return SymbolsOrErr.takeError();
 
     offloading::SYCLImage Image;
-    Image.Image = std::move(*ImageOrErr);
+    Image.Image = offloading::LazyMemoryBuffer(row.getCell("Code"));
```

Comment:
Will [`llvm::getLazyIRFileModule`](https://llvm.org/doxygen/namespacellvm.html#aac226baa3ffd0f255a8d2b6d978b81b2) work here?

Reply:
In JIT case, sure.
In AOT case, offload wrapper accepts device image as a byte array instead of LLVM IR Module. This is the reason why this is LazyMemoryBuffer and not Lazy LLVM Module.

Reply:
I see, thanks